apps/condo/pages/employee/[id]/index.tsx

```diff
@@ -277,7 +277,7 @@ export const EmployeeInfoPage = () => {
     const updateEmployeeAction = OrganizationEmployee.useUpdate({}, () => refetch())
     const softDeleteAction = OrganizationEmployee.useSoftDelete(() => Router.push('/employee/'))
 
-    const isEmployeeEditable = canManageEmployee(link, employee)
+    const isEmployeeEditable = get(link, ['role', 'canManageEmployees'], null) || get(link, ['role', 'canUpdateEmployees'], null)
```

Comment:
It's a bit strange to keep old and new format at the same time...
I'm 100% sure someone else will forget about it later...

So I guess you need to migrate old format to new or smth...